.jvmopts

```diff
@@ -6,6 +6,7 @@
 -XX:ReservedCodeCacheSize=512m
 -XX:+TieredCompilation
 -XX:+UseParallelGC
+-XX:+CMSClassUnloadingEnabled
```

Comment:
Why is this needed?

Reply:
This was added with `-XX:+UseConcMarkSweepGC`  to fix https://github.com/spotify/scio/pull/3361#issuecomment-702776811, but `UseConcMarkSweepGC` is obsoleted and the issue vanished after rebased with the new master. This is leftover from the cleaning, I will remove this.